Upgrade to `nu-ansi-term` 0.49

sholderbach opened the pull request:
This release fixes the problem that blocked #68

This PR just keeps the behavior as before and does not yet incorporate
the GNU compatibility focussed changes as worked on in #66

sholderbach committed:
Upgrade to `nu-ansi-term` 0.49

This release fixes the problem that blocked #68

This PR just keeps the behavior as before and does not yet incorporate
the GNU compatibility focussed changes as worked on in #66

sharkdp commented:
Looks like we need to bump MSRV, which is fine for me:

> error: package `rustix v0.38.4` cannot be built because it requires rustc 1.63 or newer, while the currently active rustc version is 1.62.1

sholderbach commented:
I can take care of that. 

The source of the MSRV increase is apparently the unrelated update from `tempfile` 3.6.0 to 3.7.0 which went from `rustix v0.37.23 -> v0.38.4`. So semver "compatible" change in the dev-dependencies.

sholderbach committed:
Increase MSRV to `1.63.0`

Due to unrelated update of dev-dependency tempfile from 3.6.0 to 3.7.0
which requires rustix 0.38.4 with increased MSRV of 1.63.0

sharkdp commented:
Thank you

sholderbach commented:
Thanks for the quick turnaorund! 

Would you mind a new release so nushell can upgrade to the latest `nu-ansi-term`?

sharkdp commented:
> Would you mind a new release so nushell can upgrade to the latest `nu-ansi-term`?

Done

sholderbach commented:
Awesome! Thanks!
